### ISSUE TITLE
Upload Annotations freezing t2wml

### ISSUE THREAD

**Issue by kyao** [usc-isi-i2/t2wml#585]
Uploading annotations is causing t2wml to freeze. Clicking on the close button does not close the pop up window.

![image](https://user-images.githubusercontent.com/1087657/128093380-cff5068f-91cd-44fb-b9b9-250e716ee470.png)

To reproduce:
+ Upload the spreadsheet `World Education Index by Country PI Meeting.xlsx` to t2wml web
+ Select `Upload Annotations` and upload the file `World_Education_Index_truncated_1628027982409.t2wmlz`

[WorldEducation.zip](https://github.com/usc-isi-i2/t2wml/files/6927325/WorldEducation.zip)

**Comment by kyao** [comment]
The original t2wmlz file in the zip file has an annotation with incorrect range. This t2wmlz file fixes the range. But, t2wml still  freezes.

[World_Education_Index_truncated_1628029457101.zip](https://github.com/usc-isi-i2/t2wml/files/6927376/World_Education_Index_truncated_1628029457101.zip)

**Comment by abhinav-kumar-thakur** [comment]
I am looking at it, will get back as soon as I get something to share.

**Comment by g1eb** [comment]
I tested this on our live demo and I think it works fine.. The "freezing" happens when the `.t2wmlz` project file with annotations is sent to the backend and waits on the response (took 9.29s to come back). But after that the upload file dialog disappeared and I can see the annotations show up, see screenshot below:

<img width="1897" alt="Screen Shot 2021-08-04 at 9 37 47 AM" src="https://user-images.githubusercontent.com/1916056/128220084-faa151a9-d7c8-4cc9-8f5d-901b123121c7.png">

**Comment by g1eb** [comment]
I can add some kind of loading animation to indicate that we're waiting on the response to come back, mitigating the "frozen" screen effect.

**Comment by kyao** [comment]
Adding the animation would be good. Also, Abhinav has a pull request addressing this issue.

**Comment by g1eb** [comment]
Created a separate ticket for the loading indicator #587 
Closing this issue
